### ISSUE TITLE
Feature Request: Allow identifying outputs with desc:

### ISSUE THREAD

**Issue by cmargiotta** [Horus645/swww#185]
Hello! 

**Problem**
On my PC, for some reason, identifiers like DP-2 or HDMI-A-1 keep changing and for this reason it is difficult to automatically set the wallpaper.

**Solution**
On Hyprland I can identify my monitors using this kind of identifiers:
`desc:Samsung Electric Company C34H89x H4ZN200636`
It would be useful to find this feature on swww too!

Anyway, thank you for this tool!

**Comment by Horus645** [comment]
Interesting, I didn't know that was possible.

**Comment by tasmo** [comment]
[Hyprpaper](https://github.com/hyprwm/hyprpaper) is using it. Would be great to see it here as well.

**Comment by grudgingly** [comment]
Is this what 70e63d7 implements?

**Comment by cplir-c** [comment]
Could you sort by the position in the output of `wlr-randr | grep -P "Position|^[^ ]+"` or something? I figure you're probably unplugging & re-plugging in displays regularly.

**Comment by cmargiotta** [comment]
No, my problem is that I use a Nvidia card and port identifiers are not preserved upon reboot
